modules/displaybooks.js

```diff
@@ -0,0 +1,33 @@
+import Store from './store.js';
+
+export default class UI {
+  static displayBooks() {
+    const books = Store.getBooks();
+    books.forEach((book) => UI.addBookToList(book));
+  }
+
+  static addBookToList(book) {
+    const list = document.querySelector('.my-books');
+    const allBooks = document.createElement('div');
+    allBooks.classList.add('mybooks-item');
+
+    allBooks.innerHTML = `
+          <p class="mybooks-p">${book.title}</p>
+          <p class="mybooks-p">By</p>
+          <p class="mybooks-p">${book.author}</p>
+          <button class="remove-book">Remove</button>
+          `;
+    list.appendChild(allBooks);
+  }
+
+  static deleteBook(element) {
+    if (element.classList.contains('remove-book')) {
+      element.parentElement.remove();
+    }
+  }
```

Comment:
- [ ] I would advise you to use the arrow function for your methods just as I have used it in the attached image, this is to show that you are updated in the usage of ES6 of js and it is an edge in the industry. Kindly apply the changes to all your methods.

![image](https://user-images.githubusercontent.com/86200356/220048679-ddad42c8-dc06-4d16-94cf-672f05036682.png)